migrations/20160912145044_setup.js

```diff
@@ -37,10 +37,18 @@ exports.up = knex =>
     table.integer('initial_resources').notNullable();
     table.integer('initial_mindset').notNullable();
     table.boolean('disabled').defaultsTo(false);
+  })
+  .createTable('event', (table) => {
+    table.increments('id');
+    table.string('name').unique().notNullable();
+    table.string('description').notNullable();
+    table.integer('default_damage').notNullable();
+    table.boolean('disabled').notNullable();
```

Comment:
To match the data model schema, (bool not required) we should default this to `false`.